lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/host.py

```diff
@@ -0,0 +1,27 @@
+from typing import Sequence
+from pxr import Usd
+
+class Host(object):
```

Comment:
Eventually, the host class should be moved to another place, not in the UI classes, another Python module.